types/stripe/index.d.ts

```diff
@@ -4688,6 +4689,12 @@ declare namespace Stripe {
              * This is the transaction number that appears on email receipts sent for this refund.
              */
             receipt_number: string;
+
+            /**
+             * Status of the refund. For credit card refunds, this can be succeeded or failed.
+             * For other types of refunds, it can be pending, succeeded, failed, or canceled.
+             */
+            status: string;
```

Comment:
The type here should really be more specific, as there are only specific strings which are valid, e.g.: `"pending" | "succeeded" | ` etc....

Reply:
Done :)